l10n/en/firefox/new/desktop.ftl

```diff
@@ -211,3 +211,12 @@ firefox-desktop-download-please-follow = Please follow <a href="{ $url }">these
 firefox-desktop-download-your-system-may-not = Your system may not meet the requirements for { -brand-name-firefox }, but you can try one of <a href="{ $url }">these versions</a>.
 
 firefox-desktop-download-taskbar = { -brand-name-firefox } will be pinned to the { -brand-name-windows } taskbar after installation
+
+# Linux
+firefox-desktop-download-linux-32 = { -brand-name-linux } 32-bit
+firefox-desktop-download-linux-64 = { -brand-name-linux } 64-bit
```

Comment:
There should be no need to repeat these strings as `download_button.ftl` is a global string file that all pages can access.

You could probably add all the strings (including the "using debian" one) to the global download button file?